kiwi/defaults.py

```diff
@@ -470,15 +470,26 @@ def get_install_volume_id():
         return 'INSTALL'
 
     @staticmethod
-    def get_snapper_config_template_file():
+    def get_snapper_config_template_file(root: str) -> str:
         """
-        Provides the default configuration template file for snapper
+        Provides the default configuration template file for snapper.
+        The location in etc/ are preferred over files in usr/
 
-        :return: file
+        :return: file path
 
         :rtype: str
         """
-        return '/etc/snapper/config-templates/default'
+        snapper_templates = [
+            'etc/snapper/config-templates/default',
+            'usr/share/snapper/config-templates/default'
+        ]
+        snapper_default_conf = ''
```

Comment:
I suggest to just return early:

```
        for snapper_template in snapper_templates:
            template_config = os.path.join(root, snapper_template)
            if os.path.exists(template_config):
                return template_config

        return None
```

Reply:
This introduces a typing issue which means the method can either return a "str" or a None type. This could be expressed with Optional[str] but I don't like this much and prefer a static type in any case.

I should just stop making stupid mistakes :/ thanks for reading all this

Reply:
In that case just:

```
        for snapper_template in snapper_templates:
            template_config = os.path.join(root, snapper_template)
            if os.path.exists(template_config):
                return template_config

        return ''
```